src/data/STATUSES/index.js

```diff
@@ -67,6 +67,9 @@ function correctIdsToMatchLogs (obj) {
 		} else {
 			status.id = status.id + STATUS_ID_OFFSET
 		}
+		if (status.stackWith) {
```

Comment:
medium; `stackWith` is unused now, so this is redundant.

Reply:
done